Source/Core/Core/HW/MemmapFunctions.cpp

```diff
@@ -853,53 +713,44 @@ static __forceinline u32 TranslatePageAddress(const u32 _Address, const XCheckTL
 
 	// hash function no 1 "xor" .360
 	u32 hash = (VSID ^ page_index);
+	u32 pte1 = bswap((VSID << 7) | api | PTE1_V);
 
 	for (int hash_func = 0; hash_func < 2; hash_func++)
 	{
+		// hash function no 2 "not" .360
 		if (hash_func == 1)
 		{
-			// hash function no 2 "not" .360
 			hash = ~hash;
+			pte1 |= PTE1_H << 24;
```

Comment:
I don't think that works because many of our bswap calls actually go to inline asm, depending on the platform